elide-datastore/elide-datastore-aggregation/pom.xml

```diff
@@ -104,6 +104,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
```

Comment:
What requires this?

Reply:
avatica-core

Reply:
changed scope to runtime to match the dependencies of calcite

Reply:
We don't use calcite to make HTTP connections though.  Is there any reason we can't simply exclude it?

Reply:
No.  If we mention `calcite` without any mention of `httpclient` the downstream will pull in the `calcite` old version.  They have the option to exclude it, but sadly `maven` does not let us exclude on their behalf.

Reply:
Can we add a comment:

Avoid CVE-2020-13956